### ISSUE TITLE
WowUP support

### ISSUE THREAD

**Issue by lgrano** [cr4ppy/Dung#3]
Overwolf has disabled the curseforge API, now addon managers like nexus, minion or WowUP wont be able to download from curse anymore. Personally id like to avoid installing anything by overwolf on my computer.
When trying to install via github url, wowup says it require a zip release with a -bc suffix, would you consider github releases for wowup?
Wowup also supports  wowinterface or Wago, as alternative.

**Comment by cr4ppy** [comment]
Yes this could be done of course, I will look into it further, thanks for the feedback

**Comment by cr4ppy** [comment]
We're now good to go thanks
